src/types.ts

```
@@ -181,13 +181,8 @@ export interface DownloadJob {
 // it stays a plain interface.
 
 export type {
-  CatalogCategory,
-  CatalogRegistryEntry,
-  CatalogChart,
-  CatalogHeader,
-  CatalogData,
-  CatalogInstall,
-  CatalogInstallsMap
+    CatalogCategory, CatalogChart, CatalogData, CatalogHeader, CatalogInstall,
```

ESLint

[error] Replace `··CatalogCategory,·CatalogChart,·CatalogData,·CatalogHeader,` with `CatalogCategory,⏎··CatalogChart,⏎··CatalogData,⏎··CatalogHeader,⏎·`

(prettier/prettier)